string profileid
string scantopic
----
bool success